### DISCUSSION TITLE
Welcome to the Human Readable Transaction Monolithic Snap!

### DISCUSSION THREAD

**Discussion by marcocastignoli** [sourcifyeth/sourcify-metamask-snap#1] [Announcements]
## Human Readable Transaction ➡️ HRT

For now we can use this repo to group all the HRT related snaps into one, remember to switch on `develop`

**Comment by marcocastignoli** [comment]
Check this: https://github.com/sourcifyeth/sourcify-metamask-snap/blob/develop/packages/snap/src/index.ts

In the [/packages/snap/src/insights/](https://github.com/sourcifyeth/sourcify-metamask-snap/tree/develop/packages/snap/src/insights) folder there are two example of "insights", you can add yours there.

Check also this idea: https://github.com/sourcifyeth/sourcify-metamask-snap/discussions/2

**Reply by EvgenyNasretdinov** [reply]
Great! I will polish and open source the code for chainprompter snap soon, and then will add it to this repo too. But still it will be an early use-case for a prototype version of our API,  so we will need to discuss in the future how we will publish the updates. To not copy paste the code every time and have a single source of truth :)
